src/Microsoft.Identity.Web.Certificateless/PodIdentityClientAssertion.cs

```diff
@@ -10,13 +10,14 @@
 namespace Microsoft.Identity.Web
 {
     /// <summary>
-    /// Gets a signed assertion from PodIdentity when an app is running in a container
-    /// in Azure Kubernetes Services. See https://aka.ms/ms-id-web/certificateless.
+    /// Gets a signed assertion from Azure workload identity for kubernetes when an app is running in a container
+    /// in Azure Kubernetes Services. See https://aka.ms/ms-id-web/certificateless and
+    /// https://learn.microsoft.com/azure/aks/workload-identity-overview
     /// </summary>
     public class PodIdentityClientAssertion : ClientAssertionProviderBase
```

Comment:
and this class can rename the same name?

Reply:
Done. Thanks!